### DISCUSSION TITLE
CS0006 Could not found \bin\Debug\net472\***.dll

### DISCUSSION THREAD

**Discussion by Smartisa** [NuGet/Home#11682] [Q&A] [answered]
I git clone "NuGet". When I run the test file: "PackageGraphAnalysisUtilitiesTests.cs", there are many code problems in CS0006. How can I rebuild the project to eliminate such errors?
Thanks!!!!!
![image](https://user-images.githubusercontent.com/47079990/159103963-65acfeb7-b895-4da5-ab54-0cb6e5c1521a.png)

**Comment by nkolev92** [comment, marked as answer]
Did you follow the build instructions https://github.com/NuGet/NuGet.Client/blob/dev/CONTRIBUTING.md#build?

**Reply by Smartisa** [reply]
Thanks a lot!
